tests/nectar/test_haf.py

```
@@ -0,0 +1,187 @@
+    haf.set_api("https://api.syncad.com/")
+    assert haf.get_current_api() == "https://api.syncad.com"
+
+
+def test_get_available_apis():
+    haf = HAF()
+    apis = haf.get_available_apis()
+    assert isinstance(apis, list)
+    assert all(api.startswith("http") for api in apis)
+
+
+def test_reputation_success(patch_requests):
```

Ruff (0.12.2)

Unused function argument: `patch_requests`

(ARG001)